src/app.js

```diff
@@ -7,12 +7,14 @@ import graphqlPlayground from 'graphql-playground-middleware-express';
 import { graphqlSchema, graphqlResolvers } from './graphql';
 import { get404HTML } from './utilities';
 import config from './constants/config';
+import cors from 'cors';
```

Comment:
ESLint is sad!

<p align="center">
  <img src="https://user-images.githubusercontent.com/11808903/34312718-c48329c2-e76e-11e7-8142-4a9e1f4f8aab.png" />
</p>

Could you please place this absolute import **above** the relative imports (right below the `body-parser` import would be great) 👍